src/types/drupal/node.ts

```diff
@@ -160,7 +168,7 @@ export interface NodeVetCenter extends DrupalNode {
   field_last_saved_by_an_editor?: string
   field_office_hours: FieldOfficeHours[]
   field_official_name: string
-  field_operating_status_facility: string
+  field_operating_status_facility: FacilityOperatingStatusFlags
```

Comment:
@laflannery, @randimays, or @chriskim2311: Do you know if this is accurate? Given that it has the same field name, I _assume_ it'll share the same values as the VAMC facility's `field_operating_status_facility`. 🤷

Reply:
Yes, @cvalarida Vet Centers and VAMC facilities will use the same values for operating status.